curp/src/server/curp_node.rs

```diff
@@ -289,7 +289,7 @@ impl<C: 'static + Command> CurpNode<C> {
             ))
         };
 
-        run_gc_tasks(Arc::clone(&cmd_board), Arc::clone(&spec_pool));
+        run_gc_tasks(Arc::clone(&cmd_board), Arc::clone(&spec_pool), &curp_cfg);
```

Comment:
We can pass the gc_interval to the run_gc_tasks directly. Hence, we don't need to wrap the curp_cfg by Arc.

Reply:
Will there be more gc related params needed in the future? If it won't, we can simply pass the interval value.

Reply:
From the current situation, there will be only one param in this gc task.

Reply:
> Will there be more gc related params needed in the future? If it won't, we can simply pass the interval value.

From the current situation, there will be only one param in this gc task.